plugins/module_utils/netbox_utils.py

```diff
@@ -1144,6 +1144,20 @@ def _update_netbox_object(self, data):
             serialized_nb_obj["tags"] = set(serialized_nb_obj["tags"])
             updated_obj["tags"] = set(data["tags"])
 
+        # Ensure idempotency between version 2.11 and 3.0
+        if (
+            serialized_nb_obj.get("latitude")
+            and data.get("latitude")
+            and self._version_check_greater("3.0", self.version)
+        ):
+            updated_obj["latitude"] = str(data["latitude"])
+        if (
+            serialized_nb_obj.get("longitude")
+            and data.get("longitude")
+            and self._version_check_greater("3.0", self.version)
```

Comment:
Instead of calling the `_version_check_greater()` function twice, assign the result of the function call to a local variable and use it for comparison

Reply:
That's a valid point. If you agree that these sort of issues can be dealt with this way I'll do something similar for the virtual machine and its vcpu argument which changed from str to float as well.